modules/custom-css/custom-css/js/core-customizer-css-preview.js

```diff
@@ -0,0 +1,47 @@
+// Originally based on https://raw.githubusercontent.com/xwp/wp-custom-scss-demo/master/custom-scss-demo-preview.js
+/* globals jpCustomizerCssPreview */
+(function( api, $ ) {
+	if ( api.settingPreviewHandlers ) {
```

Comment:
This will be always empty in WordPress<4.7. Since Jetpack supports WordPress≥4.5, perhaps this entire `core-customizer-css-preview.js` file should be skipped from being enqueued at all, and the `custom_css` setting should always get a `refresh` transport.